scipy/stats/tests/test_morestats.py

```diff
@@ -1253,6 +1253,15 @@ def test_mood_alternative(self):
         with pytest.raises(ValueError, match="alternative must be..."):
             stats.mood(x, y, alternative='ekki-ekki')
 
+    @pytest.mark.parametrize("alternative", ['two-sided', 'less', 'greater'])
+    def test_result(self, alternative):
+        np.random.seed(1234)
+        x1 = np.random.randn(10, 1)
+        x2 = np.random.randn(15, 1)
```

Comment:
We should use the generator interface as this is deprecated.
```suggestion
        rng = np.random.default_rng('265827767938813079281100964083953437622')
        x1 = rng.standard_normal((10, 1))
        x2 = rng.standard_normal((15, 1))
```